views/playground/responsive-utility-classes/index.html.twig

```diff
@@ -0,0 +1,70 @@
+{% extends '@pulsar/pulsar/layouts/base.html.twig' %}
```

Comment:
Would this playground be more appropriate as a new section in the Lexicon and GitBook?